flogger/flogger-api/src/test/kotlin/io/spine/logging/flogger/LogContextSpec.kt

```diff
@@ -52,7 +36,23 @@ import io.kotest.matchers.shouldBe
 import io.kotest.matchers.shouldNotBe
 import io.kotest.matchers.throwable.shouldHaveMessage
 import io.kotest.matchers.types.shouldNotBeSameInstanceAs
+import io.spine.logging.flogger.DurationRateLimiter.newRateLimitPeriod
```

Comment:
We'll need to sit down and review the next step of the changing the packaging structure, so that we avoid `flogger` entirely. Of course, let's finish the current plans first.